client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java

```diff
@@ -717,6 +717,35 @@ public void shouldRedactAuthorizationHeaderInLogsAndExceptionMessage() throws Ex
         server.disposeNow();
     }
 
+    @Test
+    public void shouldRedactSensitiveHeaderInLogsAndExceptionMessage() throws Exception {
+        DisposableServer                   server       = startServer(BAD_REQUEST, "someError");
+        HttpClientConfig                   config       = new HttpClientConfig("localhost:" + server.port());
+        Map<String, String>                headers      = new HashMap<>();
+        headers.put("Cookie", "pepperidge farm");
+        config.setDevHeaders(headers);
+        TestResource                       resource     = getHttpProxy(config);
+
+        HttpClient.markHeaderAsSensitive(resource, "cookie");
+
+        assertThatExceptionOfType(WebException.class)
```

Comment:
FYI we now have:

ObservableAssertions in the framework where we can do observable assertions.
ObservableAssertions.assertThatExceptionOfType(NoSuchElementException.class)
			.isEmittedBy(observableWithoutToBlockingSingle)